mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -70,7 +95,8 @@ void mlir::populateArithToEmitCPatterns(TypeConverter &typeConverter,
     ArithOpConversion<arith::AddFOp, emitc::AddOp>,
     ArithOpConversion<arith::DivFOp, emitc::DivOp>,
     ArithOpConversion<arith::MulFOp, emitc::MulOp>,
-    ArithOpConversion<arith::SubFOp, emitc::SubOp>
+    ArithOpConversion<arith::SubFOp, emitc::SubOp>,
+    SelectOpConversion
```

Comment:
```suggestion
    ArithSelectOpConversion
```
Maybe rename it such that is more consistent to the rest?

Reply:
The intend behind the naming `ArithOp` is that this is a conversion for a "genric" arith op. The `ArithToEmitC` already indicates that this file only contains conversions from the Arith to the EmitC dialect.